client/reader/following/intro.jsx

```diff
@@ -0,0 +1,64 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import { hasGraduatedRecommendations } from 'state/reader/start/selectors';
+import { requestGraduate } from 'state/reader/start/actions';
+
+const FollowingIntro = ( props ) => {
+	if ( ! props.isNewReader ) {
+		//return null;
+	}
+
+	const handleIntroClose = () => {
+		props.requestGraduate();
+	};
+
+	return (
+		<header className="following__intro">
```

Comment:
I don't think I've ever seen this style before of adding a newline after each tag